M2/libraries/msolve/Makefile.in

```diff
@@ -1,5 +1,5 @@
 HOMEPAGE = https://msolve.lip6.fr/
-VERSION = 0.7.0
+VERSION = 0.7.1
```

Comment:
Maybe just tie this to the latest release? You can get the url or just the version with:
```
curl -s https://api.github.com/repos/Algebraic-Solving/msolve/releases/latest | jq .tarball_url
curl -s https://api.github.com/repos/Algebraic-Solving/msolve/releases/latest | jq .name
```

Reply:
> I'm a little hesitant to do this just in case a potential future release changes behavior enough to break the package.

We will find out about this anyway, because the brew version will update anyway and github actions will fail if that happens!

> There probably should be manual verification that the latest version indeed works.

But we don't check the version of msolve in configure, so it could be too small or too large, so what's the point?

> All of the other libraries/programs in the autotools build also have hard-coded versions.

Not sure if this logic applies equally to something like factory or flint vs a software having rapid development.

Reply:
You make excellent points and I think I agree with you now that this would be a good idea.  (For example, #3367 is proof that we've done a poor job of keeping track of these things!)

Perhaps in another PR, though.  It would probably be good to generalize things so there would be some common latest-version-finding Makefile code that could be used by any of the programs and libraries.